contactsummary/js/src/contact-summary/services/contract.js

```diff
@@ -89,13 +89,31 @@ define([
 
     /**
      * @ngdoc method
-     * @name getContracts
+     * @name resetContracts
      * @methodOf ContractService
      * @returns {*}
      */
+    factory.resetContracts = function () {
+      contracts=[];
+      factory.collection = {
```

Comment:
Where does the structure of this object comes from (`insertItem`, `getItem`, etc)